### DISCUSSION TITLE
RSS feeds do not update automatically

### DISCUSSION THREAD

**Discussion by davide1295** [FreshRSS/FreshRSS#4303] [Q&A]
Hello,
I have a problem with the automatic update of the feed. I have several rss sources but only one of it updates automatically, the others to read new articles I have to manually select update source.
How can I fix this behavior?
Thank you

**Comment by Frenzie** [comment]
In principle the best method is with a cron job, see https://freshrss.github.io/FreshRSS/en/admins/08_FeedUpdates.html

Alternatively: https://freshrss.github.io/FreshRSS/en/users/03_Main_view.html#online-cron

**Comment by davide1295** [comment]
I followed the instructions in this link: https://freshrss.github.io/FreshRSS/en/admins/08_FeedUpdates.html
but it still doesn't work. Only one in 5 feeds are automatically updated.

**Reply by Frenzie** [reply]
They won't update if they were updated recently to prevent DDoSing the server of the feed. You can adjust the specifics both globally and per feed.

**Reply by Alkarex** [reply]
The feeds refresh frequencies are defined at feed level and at FreshRSS level in the configuration. To be sure, wait at least a couple of hours before trying to update the feeds.
If there is still a problem, check your FreshRSS logs in `./FreshRSS/data/users/*/log*.txt` and the output of your cron job (e.g. `/tmp/FreshRSS.log`),

**Reply by davide1295** [reply]
The .log file contains only this string:

_[Fri, 01 Apr 2022 10:53:54 +0200] [warning] --- Error while WebSub subscribing to https://www.xxx.com/feed/gn_
